boards/arm/olimexino_stm32f303rc/doc/index.rst

```diff
@@ -0,0 +1,126 @@
+.. _olimexino_stm32f303rc_board:
+
+Olimexino STM32F303RC
+################
+
+Overview
+********
+
+The Olimexino STM32F303RC board features an ARM Cortex-M4 based STM32F303RC
+mixed-signal MCU with FPU and DSP instructions capable of running at 72 MHz.
+Here are some highlights of the Olimexino STM32F303RC board:
+
+- STM32 microcontroller in LQFP64 package
+- LSE crystal: 32.768 kHz crystal oscillator
+- Flexible board power supply:
+
+  - 5 V from USB VBUS
+  - External power sources: 9 - 30 V
+
+- Two user LED
+- Two push-buttons: BOOT/USER and RESET
+- SIT1050T CAN transceiver
+- UEXT connector
+- SWD (2x5 1.27mm) connector
+- Arduino* Uno V3 connectors
+
+More information about the board can be found at the `Olimexino STM32F303RC website`_.
+Hardware
+********
+
+The Olimexino STM32F303RC provides the following hardware components:
+
+- STM32F303RCT6 in QFP64 package
+- ARM |reg| 32-bit Cortex |reg| -M4 CPU with FPU
+- 72 MHz max CPU frequency
+- VDD from 2.0 V to 3.6 V
+- 256 KB Flash
+- 40 KB SRAM
+- Routine booster: 8 Kbytes of SRAM on instruction and data bus
+- GPIO with external interrupt capability
+- 4x12-bit ADC with 39 channels
+- 2x12-bit D/A converters
+- RTC
+- General Purpose Timers (13)
+- USART/UART (5)
+- I2C (2)
+- SPI (3)
+- CAN
+- USB 2.0 full speed interface
+- Infrared transmitter
+- DMA Controller
+
+More information about the STM32F303RC can be found here:
+
+- `STM32F303RC on www.st.com`_
+- `STM32F303RC reference manual`_
+- `STM32F303RC datasheet`_
+
+Supported Features
+==================
+
+The default configuration can be found in the defconfig file:
+``boards/arm/olimexino_stm32f303rc/olimexino_stm32f303rc_defconfig``
+
```

Comment:
Default configuration is nice but this is pretty much the same for every board, and quite basic.
I'd add a reference to the configured peripherals that could be seen in board's device tree file